### PR TITLE
add 8.1 in basic features

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Update basic features for v8.1:

- Remove v4.0, v5.0, v7.6, and v8.0 columns
- Add the v8.1 column
- Runaway Queries management becomes GA in v8.1 (ref https://github.com/pingcap/docs-cn/pull/17209)

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md
Comment (Oreoxmt) [inline basic-features.md:1]: Preview: https://pingcap-docsite-preview-git-v81-preview-oreoxmts-projects.vercel.app/zh/tidb/stable/basic-features

**Review by Frank945946 — APPROVED** [review]

File: basic-features.md [hunk 1]
```diff
@@ -20,245 +20,245 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [DM WebUI](/dm/dm-webui-guide.md) | E | E | E | E | E | N | N | N | N |
+| [前台限流](/tikv-configuration-file.md#前台限流) | Y | Y | Y | Y | E | N | N | N | N |
+| [后台限流](/tikv-configuration-file.md#后台限流) | E | E | E | E | N | N | N | N | N |
+| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore) | Y | Y | Y | Y | N | N | N | N | N |
+| [PITR](/br/br-pitr-guide.md) | Y | Y | Y | Y | N | N | N | N | N |
+| [全局内存控制](/configure-memory-usage.md#如何配置-tidb-server-实例使用内存的阈值) | Y | Y | Y | Y | N | N | N | N | N |
+| [RawKV 跨集群复制](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | E | E| E | E | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | E |
+| [资源管控 (Resource Control)](/tidb-resource-control.md) | Y | Y | Y | N | N | N | N | N | N |
+| [Runaway Queries 自动管理](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries) | E | E | N | N | N | N | N | N | N |
```
Comment (lilin90) [inline basic-features.md:253]:
```suggestion
| [Runaway Queries 自动管理](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries) | Y | E | N | N | N | N | N | N | N |
```

**Review by lilin90 — COMMENTED** [review]
Ref: https://github.com/pingcap/docs-cn/pull/17209

**Review by songrijie — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Pull request merged**